README.md

```diff
@@ -54,6 +54,32 @@ Starting from zone.js v0.8.9, you can choose which web API module you want to pa
 For more details, please
 see [MODULE.md](MODULE.md).
 
+## Bundles
+There are several bundles under `dist` folder.
+- `zone.js`, the default bundle, contains the most used APIs such as `setTimeout/Promise/EventTarget...`, also this bundle will support all evergreen and legacy (IE/Legacy Firefox/Legacy Safari) Browsers
+- `zone-evergreen.js`, the bundle for evergreen browsers, will not include the `patch` for `legacy` browsers such as `IE` or old versions of `Firefox/Safari`.
+- `zone-legacy.js`, the patch bundle for legacy browsers, will only include the `patch` for `legacy` browsers such as `IE` or old versions of `Firefox/Safari`. Need to work with `zone-evergreen.js`.
+- `zone-testing.js`, the bundle for zone testing support, including `jasmine/mocha` support and `async/fakeAsync/sync` test utilities.
+- `zone-externs.js`, the API definitions for `closure compiler`.
+- Specified API patch bundles
+  1. `webapis-media-query.js`, patch for `MediaQuery APIs`. 
+  2. `webapis-notification.js`, patch for `Notification APIs`. 
+  3. `webapis-rtc-peer-connection.js`, patch for `RTCPeerConnection APIs`. 
+  4. `webapis-shadydom.js`, patch for `Shady DOM APIs`. 
+  5. `zone-bluebird.js`, patch for `Bluebird APIs`. 
+  6. `zone-error.js`, patch for `Error Global Object`, support remove `Zone StackTrace`. 
+  7. `zone-patch-canvas.js`, patch for `Canvas API`.
+  8. `zone-patch-cordova.js`, patch for `Cordova API`.
+  9. `zone-patch-electron.js`, patch for `Electron API`.
+  10. `zone-patch-fetch.js`, patch for `Fetch API`.
+  11. `zone-patch-jsonp.js`, utility for `jsonp API`.
```

Comment:
Utility? Why not "patch"?

Reply:
Thanks, the `zone-patch-jsonp` here is not really a patch, because `jsonp` is not a `DOM API` but only a `mechianism`, so here in `zone-patch-jsonp.js` we only provide a `utility method` to help user create their own patch for `jsonp`.